apiary.apib

```diff
@@ -2361,6 +2361,7 @@ Get a list of invoices.
                     + due (Money)
                 + created_at: `2016-02-04T16:44:33+00:00` (string)
                 + updated_at: `2016-02-05T16:44:33+00:00` (string)
+                + `web_url`: `https://app.teamleader.dev/invoice_detail.php?id=25`
```

Comment:
Should it be a uuid?

Reply:
Yes, invoice_detail.php can handle uuid's already, so let's use uuid's.

Reply:
Okay

Reply:
okay